.gitignore

```diff
@@ -129,4 +129,3 @@ dmypy.json
 
 # Development
 .idea/
-build/
```

Comment:
The build folder is meant to be ignore by git

Reply:
there is 2 build/ in the gitignore file

Reply:
Ignore file the same as the main ignore file